pennylane_lightning/core/_state_vector_base.py

```diff
@@ -35,6 +35,7 @@ class LightningBaseStateVector(ABC):
         num_wires(int): the number of wires to initialize the device with
         dtype: Datatypes for state-vector representation. Must be one of
             ``np.complex64`` or ``np.complex128``. Default is ``np.complex128``
+        sync Optional(bool): immediately sync with host-sv after applying operation.
```

Comment:
Synchronized host-device data copy.

Reply:
Currently, I will remove this line because the base class not longer accept this argument.